### DISCUSSION TITLE
Grid Multiple Selection When load Page.

### DISCUSSION THREAD

**Discussion by ronaldsantamaria** [vikramlearning/blazorbootstrap#387] [General]
Hello,

Can we check a Row from when we load the page using the grid a multiple selection?

Thanks.

**Comment by gvreddy04** [comment]
Thanks for trying **BlazorBootstrap**. Do you want to preselect a row once the grid is rendered? Is this correct understanding?

**Comment by ronaldsantamaria** [comment]
Yes  before rendered, I did a alternative solution using your example , thanks

https://github.com/vikramlearning/blazorbootstrap-examples
